src/formatDate.js

```diff
@@ -8,7 +8,31 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateParts = date.split(fromFormat[3]);
```

Comment:
You're using the last element of the 'fromFormat' array to split the date string. It would be clearer to name this element, for example, 'fromSeparator', to improve code readability.